style.css

```diff
@@ -347,3 +380,471 @@ span {
   display: flex;
   flex-direction: column;
 }
+
+@media screen and (min-width: 992px) {
+  body {
+    margin: 0;
+  }
+
+  .headers {
+    background-image: url(bgimages/twocirclesheader.png);
+    background-repeat: no-repeat;
+    display: flex;
+  }
+
+  .nav-buttons {
+    background-color: #e5e5e5;
+    display: flex;
+    display: block;
+    justify-content: flex-end;
+    list-style-type: none;
+    color: inherit;
+    border: none;
+    padding: 0;
+    font: inherit;
+    cursor: pointer;
+    outline: inherit;
+  }
+
+  .recent {
+    display: none;
+  }
+
+  .mobile {
+    display: none;
+  }
+
+  .bars {
+    display: none;
+  }
+
+  .contact {
+    display: none;
+  }
+
+  .mobile-v {
+    display: none;
+  }
+
+  .icons {
+    margin-left: 70px;
+    margin-right: 100px;
+    padding-bottom: 30px;
+    margin-bottom: 20px;
+    margin-top: 28px;
+    display: flex;
+    gap: 15px;
+    justify-content: center;
+    height: 16px;
+    width: 20px;
+  }
+
+  .nav-li {
+    display: flex;
+    gap: 20px;
+    background-color: #191765;
+    padding-right: 5rem;
+  }
+
+  .headers h1 {
+    font-style: normal;
+    font-weight: 700;
+    font-family: "Roboto", sans-serif;
+    font-size: 90px;
+    line-height: 105px;
+    text-align: center;
+    color: #9391ff;
+    margin: 24px 0;
+    width: 1156px;
+    height: 96px;
+  }
+
+  .headers h2 {
+    width: 1156px;
+    height: 72px;
+    left: 0;
+    top: 120px;
+    font-family: "Roboto", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 40px;
+    line-height: 47px;
+    text-align: center;
+    color: #dfe1e6;
+    margin: 24px 0;
+  }
+
+  .headers p {
+    position: static;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    font-family: "Roboto", sans-serif;
+    text-align: center;
+    padding: 0 0;
+    width: 447px;
+    height: 155px;
+    line-height: 32px;
+    color: #b3bac5;
+    margin: 24px 0;
+  }
+
+  .line {
+    color: #646369;
+    height: 1px;
+    width: 448px;
+    margin-top: 50px;
+  }
+
+  .software {
+    display: flex;
+    margin-top: 75px;
+    flex-wrap: wrap;
+    gap: 15px;
+  }
```

Comment:
- [ ] Please make sure to maintain the design for the mobile version as close as possible to the Figma design. As you can see your mobile version is different than the one in the Figma template please fix this as this is one of the requirements for this project. :+1: 

Your Implementation | Figma Design
:---:|:---:
![image](https://user-images.githubusercontent.com/28109626/149584053-84ec9d6d-76f7-4f86-aae8-0113f0893a24.png) | ![image](https://user-images.githubusercontent.com/28109626/149583973-f1340207-5e1d-4188-8706-6083b700a662.png)